src/QMCWaveFunctions/Jastrow/PadeFunctors.h

```diff
@@ -427,7 +427,11 @@ struct Pade2ndOrderFunctor:public OptimizableFunctorBase
     while(tcur != NULL)
     {
       std::string cname((const char*)(tcur->name));
-      if(cname == "parameter" || cname == "Var")
+      if(cname == "parameter")
+      {
+        throw std::runtime_error("Keyword 'parameter' has been replaced by 'var'");
```

Comment:
This should give some idea of the location in the XML file where change is needed - parameter is not being replace by var everywhere.  Maybe it could read "Keyword 'parameter' has been replaced by 'var' in jastrow/correlation" ?

Reply:
That is a good point.